package.json

```diff
@@ -11,8 +11,8 @@
   },
   "scripts": {
     "build-ui": "lerna run build --scope navi-app --stream",
-    "start": "cd packages/webservice && ./gradlew bootRun",
     "postinstall": "lerna bootstrap --concurrency 2 --ci",
+    "start": "cd packages/webservice && ./gradlew bootRun",
```

Comment:
Do we need this change?

Reply:
i dont think the order matters. it must be resulting from one of my changes that i undid. i can rever to previous order.